packages/cli/src/commands/ts.ts

```diff
@@ -33,9 +33,6 @@ const DEFAULT_TS_BUILD = "beta";
 let TSC_PATH = "";
 let TS_MIGRATE_PATH = "";
 
-// cwd for development only
```

Comment:
@lynchbomb why was this here in the first place?

Reply:
So we can inspect the file report during dev at various states before its compiled output. We use tmp dir so those steps are lost otherwise.